### ISSUE TITLE
IE 11 use-strict

### ISSUE THREAD

**Issue by yosh970** [franciscop/translate#11]
Hey there, using this awesome lib thats helps so much !

How ever I encounter some problems with IE 11.

I get a blank page when i run my app and get an error : 

> multiple definitions of a property not allowed in strict mode

It seems to me that it comes from translate, as you can see on the attached file.

I might be wrong tho, but if not, any idea how to solve this problem?

Thanks a lot !
![capture2](https://user-images.githubusercontent.com/29760820/52494602-b74c5080-2bce-11e9-91c6-ff3403c2793a.JPG)

**Comment by franciscop** [comment]
Interesting issue, I seem to have found the issue and fixed it (some repeated keys in the data folder), please upgrade to `1.0.4` and let me know if the issue persists! :smile:

**Comment by yosh970** [comment]
hello, thank you for the patch, it partially solved the error, however i now get a syntax error still pointing to the package translate.

Really hard to debug with IE console, but i managed to get into the debugger to join this capture.

Do you think it comes from your source code, or from the way i implemented the package?

Thanks a lot for your precious help !

![capture10](https://user-images.githubusercontent.com/29760820/52557748-a11dda80-2df0-11e9-970c-77805a83e650.JPG)

**Comment by franciscop** [comment]
This time it seems you are out of luck, I am not using anything like that: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Deprecated_caller_or_arguments_usage

I do not know though why your code ends up with an `eval("...")`, I don't know if that's normal webpack behavior or not. And with this limited information I cannot do much more, I haven't used and cannot find `.caller`, `arguments`, etc. in this library code. Are you using Create-React-App? Since 2.0 the browser support changed, so please review it. Also, for IE make sure you are including a polyfill as in the instructions (but that doesn't seem to be the error).

To find out better whether the issue comes from `translate` or your code, could you try defining translate as this please?

```js
// Comment this out:
// import translate from 'translate';
const translate = () => 'Hello world';
```

Compile and try again in IE to see if the issue comes from this library or from your code/webpack.

**Comment by yosh970** [comment]
I defined translate as you suggested, this console log gives no output in IE console.

I installed **es6-promise** and **isomotfic-fetch** and used them in my main.js such as : 

`require('es6-promise').polyfill()`
`require('isomorphic-fetch')`

but this doesn not solve the problem.

Also installed **babel-polyfill** and set it as entry of my webpack-base-conf such as : 
`
app: ['babel-polyfill', 'isomorphic-fetch', './src/main.js']
`
but again not working.


I still endup with the error : 

> Syntax error

 (image attached).

Any idea mate?

![capt](https://user-images.githubusercontent.com/29760820/52645830-db1ad980-2ee1-11e9-87f7-089b98c947b9.JPG)

**Comment by franciscop** [comment]
I'm a bit unsure about `require()` vs `import`, but that shouldn't be an issue. Could you share a bit of the code of how you are using it please?

Meanwhile let me reopen it, tag it as "help needed" and see if I can get an OSS account in BrowserStack to try to debug it.

**Comment by yosh970** [comment]
Desperate times call for desperate measures.

To quickly solve the problem I downloaded the minified version of you package (translate.min.js).

Then i [Babelyfied](https://babeljs.io/repl) it.

Once that done, i imported the file in my app under tool directory, updated all the imports with the new path (import translate from 'my/directory/translate.js').

I disabled the linter in that file.

IE could not resolve includes, luckly there was only one include, so i replaced it in the if statement by `iso.indexOf(name) === -1`

I know it is not a good way of handling problems, but in the end it works.. so far. But at least I didnt not have to rethink all my webpack / Babel setup.

Thank you for your help and your patience.
